<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgParam</class>
 <widget class="QDialog" name="DlgParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit_param"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_valider">
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_annuler">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
